packages/node_modules/@ciscospark/internal-plugin-metrics/src/metrics.js

```diff
@@ -15,6 +16,11 @@ const Metrics = SparkPlugin.extend({
     callDiagnosticEventsBatcher: CallDiagnosticEventsBatcher
   },
 
+  metricMap: new Map(),
```

Comment:
Wouldn't we want to share the context information across instances?

I could conceive of situations where we'd want to share the information across different modules that may not necessarily be talking to the same instance.

I believe in the case of the web-client, we do have a single instance of Spark-JS-SDK, but seems it might be useful.

Is the thinking this is not expected behavior from most people; e.g.: if I create a new instance, I expect all the context information to be fresh, and that the instance does not consider the other metrics I've started to create for the purposes of sharing between the SDK and the client?

Reply:
the metrics plugin will be instantiated once for each sdk instance. Typically, you would only make a second sdk instance if you had a different set of credentials (e.g. for a node web server that handles multiple user requests). In those cases, you definitely don't want to share the metrics map across instances.